spec/test-data/translations/glotpress-download/expected/values-zh-rCN/strings.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Translation-Revision-Date: 2021-03-16 18:54:04+0000
+Plural-Forms: nplurals=1; plural=0;
+Generator: GlotPress/2.4.0-alpha
+Language: zh_CN
```

Comment:
To ensure we test at least one non-roman language, so that we validate that handle unicode properly.